### PR TITLE
Add AWSServiceRoleForVPCTransitGateway Rules Directly

### PR DESCRIPTION
**Pull request by dustman9000**
Follow-up PR to https://github.com/openshift/aws-account-operator/pull/528 to add rules directly.

Reference: https://docs.aws.amazon.com/vpc/latest/tgw/tgw-service-linked-roles.html

### CODE REVIEW

**Comment by mwoodson** [comment]
+1 I approve this

File: deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml [hunk 1]
```diff
@@ -32,6 +32,11 @@ spec:
           - "ec2:DeleteVpnConnectionRoute"
           - "ec2:ModifyVpcPeeringConnectionOptions"
           - "ec2:RejectVpcPeeringConnection"
+          - "ec2:CreateNetworkInterface"
+          - "ec2:DescribeNetworkInterface"
+          - "ec2:ModifyNetworkInterfaceAttribute"
+          - "ec2:DeleteNetworkInterface"
+          - "ec2:CreateNetworkInterfacePermission"
```
Comment (rogbas) [inline deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml:39]: @dustman9000 don't you need to add  vpc:TransientSomething permissions?

**Comment by billmvt** [comment]
ROSA Change Freeze Exception approved. This fixes #528 which had issues in Stage.

**Comment by jharrington22** [comment]
+1 to approve this.

File: deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml (hunk 1 above)
Comment (dustman9000) [reply deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml:39]: @rogbas - not that i'm aware of...do you mean "vpc:transit"? We are not using those for PrivateLink...

Comment (rogbas) [reply deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml:39]: ok then, just checking

**Pull request merged**